assembl/static2/js/app/components/debate/common/ideas.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Grid } from 'react-bootstrap';
+import _filter from 'lodash/filter';
+import IdeasLevel from './ideasLevel';
+
+class Ideas extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedIdeas: [''],
+      isInitialState: true
+    };
+    this.onSeeSubIdeasClick = this.onSeeSubIdeasClick.bind(this);
+  }
+  onSeeSubIdeasClick(ideaId, level) {
+    const updatedSelectedIdeas = this.state.selectedIdeas.slice(0, level + 2);
+    updatedSelectedIdeas[level] = ideaId;
+    updatedSelectedIdeas[level + 1] = '';
```

Comment:
The ideas level which shows at the bottom is always represented by an empty string because this means that no idea is selected in this level